lib/generators/react_on_rails/templates/bootstrap/client/assets/stylesheets/_post-bootstrap.scss

```diff
@@ -0,0 +1,6 @@
+// The Rails server imports this file via app/assets/stylesheets/application.css.scss
+// The webpack HMR dev server loads this file via client/bootstrap-sass.config.js
+
+// In either case, this file is loaded AFTER bootstrap has been loaded.
+// This is where you can add your own styles and override bootstrap styles
+// utilizing bootstrap variables and mixins.
```

Comment:
@justin808 I realize this is a very arbitrary time to draw a line, but I think we should avoid making the generator put *too* much stuff in there. Then again, maybe this is a place that warrants it. Not sure :/